src/quacc/atoms/skzcam.py

```diff
@@ -13,12 +13,489 @@
 from monty.io import zopen
 from monty.os.path import zpath
 
+
+class ElementInfo(TypedDict):
+    core: int
+    basis: str
+    ecp: str
+    ri_scf_basis: str
+    ri_cwft_basis: str
+
+
 if TYPE_CHECKING:
+    from ase.atom import Atom
     from numpy.typing import NDArray
 
 has_chemshell = find_spec("chemsh") is not None
 
 
+def create_orca_eint_blocks(
+    embedded_adsorbed_cluster: Atoms,
+    quantum_cluster_indices: list[int],
+    ecp_region_indices: list[int],
+    element_info: ElementInfo | None = None,
+    pal_nprocs_block: dict[str, int] | None = None,
+    method_block: dict[str, str] | None = None,
+    scf_block: dict[str, str] | None = None,
+    ecp_info: dict[str, str] | None = None,
+    include_cp: bool = True,
+    multiplicity: dict | None = None,
+) -> tuple[str, str, str]:
+    """
+    Creates the orcablocks input for the ORCA ASE calculator.
+
+    Parameters
+    ----------
+    embedded_adsorbed_cluster
+        The ASE Atoms object containing the atomic coordinates and atomic charges from the .pun file, as well as the atom type.
+    quantum_cluster_indices
+        A list containing the indices of the atoms in each quantum cluster.
+    ecp_region_indices
+        A list containing the indices of the atoms in each ECP region.
+    element_info
+        A dictionary with elements as keys which gives the (1) number of core electrons as 'core', (2) basis set as 'basis', (3) effective core potential as 'ecp', (4) resolution-of-identity/density-fitting auxiliary basis set for DFT/HF calculations as 'ri_scf_basis' and (5) resolution-of-identity/density-fitting for correlated wave-function methods as 'ri_cwft_basis'.
+    pal_nprocs_block
+        A dictionary with the number of processors for the PAL block as 'nprocs' and the maximum memory-per-core in megabytes blocks as 'maxcore'.
+    method_block
+        A dictionary that contains the method block for the ORCA input file.
+    scf_block
+        A dictionary that contains the SCF block for the ORCA input file.
```

Comment:
It wasn't immediately clear to me until looking at the test what the format should be. It should say in the docstring that the key is the ORCA setting and the value is that setting's value.